docs/registry-fields.md

```diff
@@ -123,6 +129,10 @@ Ex. `0` and `5` for Bitcoin.
 Human Readable Prefix used to prefix an address, used to indicate type of address, to minimalize risk of accidental address mix-up across chains.
 Ex. `'bc'` for Bitcoin, `'cosmos'` for Cosmos.
 
+**`chainId`**
+Chain identifier, used by forks, e.g. in case of Ethereum (a decimal number), or Cosomos (a string ID).
```

Comment:
```suggestion
Chain identifier, in case of Ethereum it's a constant decimal number; in Cosmos, it's a dynamic string network id (different after each upgrade)
Please note the chain id might not be always latest in registry.
```